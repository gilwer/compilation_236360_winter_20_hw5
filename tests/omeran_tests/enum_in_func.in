enum days { SUN, MON, TUES };

int foo() {
    enum months { JAN, FEB, MAR };
    if ((int)JAN < (int)FEB) {
        return 5*(int)MAR; // returns 10
    }
    return (int)MAR; // returns 2
}

void main() {
    int i;
    while (i < foo()) {
        i = i + 1;
        print("should print ten times");
    }
}