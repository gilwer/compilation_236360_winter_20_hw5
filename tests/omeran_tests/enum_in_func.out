should print ten times
should print ten times
should print ten times
should print ten times
should print ten times
should print ten times
should print ten times
should print ten times
should print ten times
should print ten times
